.rubocop_local.yml

```diff
@@ -0,0 +1,19 @@
+#
+# Overrides
+#
+GlobalVars:
+  AllowedVariables:
```

Comment:
They are still used in this code base, although `$vim_log` is only used in tests.  Someone should remove `$vim_log` in a separate PR.